src/Taskter.Infrastructure/Repositories/DummyRepository.cs

```diff
@@ -6,34 +6,34 @@
 
 namespace Taskter.Infrastructure.Repositories
 {
-    public class DummyRepository : IDummyRepository
-    {
-        private readonly TaskterDbContext _context;
+    // public class DummyRepository : IDummyRepository
```

Comment:
Please remove commented code. Reason for that is less lines of code to worry about.

Reply:
I also think that this whole repo can be removed (since it's commented out)

Reply:
Also also, when I comment on suggestions like this, I expect that you should go through code and remove all occurrences. Example `ProjectTaskRepository`